tox.ini

```diff
@@ -26,6 +26,7 @@ commands=
 commands =
     npm install --prefix {toxinidir}
     {toxinidir}/node_modules/.bin/jshint ui/static/ui/js ui/jstests/
+    {toxinidir}/node_modules/.bin/jsxhint 'ui/**/*.jsx' --jsx-only
```

Comment:
Yes, specifying the directory isn't enough, it sees zero files and assumes everything's ok. I needed the glob here